editor/editor_node.cpp

```diff
@@ -4400,6 +4413,8 @@ void EditorNode::_quick_opened() {
 		const String &res_path = files[i];
 		if (open_scene_dialog || ClassDB::is_parent_class(ResourceLoader::get_resource_type(res_path), "PackedScene")) {
 			open_request(res_path);
+		} else if (ClassDB::is_parent_class(ResourceLoader::get_resource_type(res_path), "ColorPalette")) {
```

Comment:
I'm not sure about that. The best option would be making a separate QuickOpen for ColorPalettes. The global QuickOpen can be used on its own (Ctrl+Alt+O) and opening a palette will have no effect.